src/applications/vaos/tests/e2e/direct-schedule.e2e.spec.js

```diff
@@ -0,0 +1,114 @@
+const Timeouts = require('../../../../platform/testing/e2e/timeouts.js');
+const VAOSHelpers = require('./vaos-helpers');
+const Auth = require('../../../../platform/testing/e2e/auth');
+
+const continueButton = '.rjsf [type="submit"]';
+
+module.exports = {
+  after: (client, done) => {
+    client.deleteCookies();
+    client.end();
+    done();
+  },
+  login: client => {
+    const token = Auth.getUserToken();
+
+    VAOSHelpers.initAppointmentListMock(token);
+
+    Auth.logIn(
+      token,
+      client,
+      '/health-care/schedule-view-va-appointments/appointments/',
+      3,
+    ).waitForElementVisible('#appointments-list', Timeouts.slow);
+    // .axeCheck('.main');
+  },
+  'Select new appointment': client => {
+    client
+      .click('#new-appointment')
+      .waitForElementVisible(continueButton, Timeouts.normal)
+      .axeCheck('.main');
+  },
+  'Choose the type of care you need': client => {
+    client.click('[value="211"]').perform(() => {
+      client
+        .click('.rjsf [type="submit"]')
+        .waitForElementPresent('h1', Timeouts.normal)
+        .assert.containsText('h1', 'Choose a VA location for your appointment');
+    });
+  },
+  'Choose a VA location for your appointment': client => {
+    client.click('[value="983"]').perform(() => {
+      client
+        .waitForElementPresent(
+          '[name = "root_vaFacility"][value = "983"]',
+          Timeouts.normal,
+        )
+        .click('[name = "root_vaFacility"][value = "983"]')
+        .pause(Timeouts.normal)
+        .click('.rjsf [type="submit"]')
+        .waitForElementPresent('h1', Timeouts.normal)
+        .assert.containsText(
+          'h1',
+          'What date and time would you like to make an appointment?',
+        );
+    });
+  },
+  'What date and time would you like to make an appointment?': client => {
+    client
+      .click(
+        '.vaos-calendar__calendars button[id^="date-cell"]:not([disabled])',
+      )
+      .perform(() => {
+        client
+          .click('.vaos-calendar__options input[id^="checkbox-0"]')
+          .click('.rjsf [type="submit"]')
+          .assert.containsText('h1', 'Reason for appointment');
+      });
+  },
+  'Reason for appointment': client => {
+    client.click('#root_reasonForAppointment_0').perform(() => {
+      client
+        .setValue('textarea#root_reasonAdditionalInfo', 'Additonal information')
+        .click('.rjsf [type="submit"]')
+        .assert.containsText('h1', 'How would you like to be seen?');
+    });
+  },
+  'How would you like to be seen?': client => {
+    client.click('input#root_visitType_0').perform(() => {
+      client
+        .click('.rjsf [type="submit"]')
+        .assert.containsText('h1', 'Contact information');
+    });
+  },
+  'Contact information': client => {
```

Comment:
I think there are some sections that are common across these tests. We typically try to put those in functions in the helpers file (you can pass `client` as a parameter) and call them from each test.

Reply:
Hey Jeff. Thanks for the review. Quick question about the comment, "there are some sections that are common across these tests.". How should this be refactored? Meaning, do you want the whole test in the helper file for the common sections?